kcbq-connector/quickstart/properties/connector.properties

```diff
@@ -39,6 +41,6 @@ transforms.RegexTransformation.replacement=$0
 project=
 # The name of the BigQuery dataset to write to (leave the '.*=' at the beginning, enter your
```

Comment:
This comment seems to be outdated now. We should update it

Reply:
Will update